x-pack/package.json

```diff
@@ -9,7 +9,7 @@
     "kbn:bootstrap": "node legacy/plugins/canvas/scripts/storybook --clean",
     "start": "gulp dev",
     "build": "gulp build",
-    "testonly": "gulp testonly",
+    "testonly": "gulp test",
     "test": "gulp test",
```

Comment:
@spalger there is still a point of keep the two different named tasks doing the same thing? Or did you miss something here?

Reply:
I just don’t want to break thing for people who have a workflow of running testonly, for now. I assume @jbudz is going to be introducing breaking changes and I don’t feel the need to pile on.

Reply:
Perhaps a good call would be: `echo testonly is deprecated; use test && gulp test`...?